src/slack_interaction_handler.ts

```diff
@@ -267,6 +262,24 @@ export async function handleVoterStatusUpdate({
         );
       }
 
+      // HACK: work around slack bug updating blocks: we must first remove the dropdown before we change it
+      const oldBlock = payload.message.blocks[2];
+      payload.message.blocks[2] = {
+        type: 'section',
+        block_id: 'asdf',
+        text: {
+          type: 'mrkdwn',
+          text: 'Updating...',
+          verbatim: false,
+        },
+      };
+      await SlackInteractionApiUtil.replaceSlackMessageBlocks({
```

Comment:
I'm not sure I follow what's going on here. I know there's a bug but why are we calling replaceSlackMessageBlocks twice?

Reply:
Updating the selected option doesn't work. So we first change the block to "Updating..." and then change it back to the drop down, which is enough to coax slack into selecting the new item.

Reply:
I see. Have you heard back from Andrew regarding this?